docs/go/how-to-continue-as-new-in-go.md

```diff
@@ -18,3 +18,9 @@ func SimpleWorkflow(ctx workflow.Context, value string) error {
 ```
 
 To check whether a Workflow Execution was spawned as a result of Continue-As-New, you can check if `workflow.GetInfo(ctx).ContinuedExecutionRunID` is not nil.
+
+## Notes
+
+- To prevent Signal loss, make sure to perform an asynchronous drain on the Signal channel. Failure to do so can result in buffered Signals being ignored and lost.
+
+- Make sure that the previous Workflow and the Continue-As-New Workflow are both referenced by the same alias. Failure to do so can cause the Workflow to Continue-As-New on an entirely different Workflow.
```

Comment:
```suggestion
- To prevent Signal loss, be sure to perform an asynchronous drain on the Signal channel. Failure to do so can result in buffered Signals being ignored and lost.
- Make sure that the previous Workflow and the Continue-As-New Workflow are referenced by the same alias. Failure to do so can cause the Workflow to Continue-As-New on an entirely different Workflow.
```